cmd/apiserver/app/server/server.go

```diff
@@ -127,3 +131,8 @@ func NewCommandServer(
 
 	return cmd, nil
 }
+
+// registerAllAdmissionPlugins registers all admission plugins
+func registerAllAdmissionPlugins(plugins *admission.Plugins) {
+	lifecycle.Register(plugins)
```

Comment:
Admission plugins registration is not global anymore, requires `*admission.Plugins` instance